gettsim/shared.py

```diff
@@ -1,6 +1,38 @@
 import inspect
 import textwrap
 
+from gettsim.config import SUPPORTED_GROUPINGS
+
+
+class KeyErrorMessage(str):
+    """Subclass str to allow for line breaks in KeyError messages"""
+
+    def __repr__(self):
+        return str(self)
```

Comment:
Codecov complains this is not being tested. Useful to add a case, in this PR or elsewhere?